public-ui/src/app/highlight-search/highlight-search.component.ts

```diff
@@ -23,4 +24,25 @@ export class HighlightSearchComponent implements OnChanges {
     }
     this.words = this.text.split(' ');
   }
+  public highlight(word) {
+    const splitWords = word.split(' ');
```

Comment:
I'm confused - the original `words` array was generated by splitting on `' '`. YOu then iterate through that list, and once again split on space here. Shouldn't this list always be length 1?